streampipes-user-management/src/main/java/org/apache/streampipes/user/management/service/TokenService.java

```diff
@@ -28,9 +28,8 @@ public class TokenService {
   public RawUserApiToken createAndStoreNewToken(String email,
                                                 RawUserApiToken baseToken) {
     UserAccount user = getUserStorage().getUserAccount(email);
-    RawUserApiToken generatedToken = TokenUtil.createToken(baseToken.getTokenName());
+    RawUserApiToken generatedToken = TokenUtil.createToken(baseToken.tokenName());
     storeToken(user, generatedToken);
-    generatedToken.setHashedToken("");
```

Comment:
Did we do this for a certain reason? From my point of view, it just does not make sense

Reply:
I think we just don't want to expose the hashed token to the user.

Reply:
Does the RawUserApiToken then even need this attribute? The only usage I can find is in a test